libs/Microsoft.MixedReality.WebRTC/LocalAudioTrack.cs

```diff
@@ -77,63 +82,70 @@ public bool Enabled
         private LocalAudioTrackInterop.InteropCallbackArgs _interopCallbackArgs;
 
         /// <summary>
-        /// Create an audio track from a local audio capture device (microphone).
+        /// Create an audio track from an existing audio track source.
+        /// 
         /// This does not add the track to any peer connection. Instead, the track must be added manually to
         /// an audio transceiver to be attached to a peer connection and transmitted to a remote peer.
         /// </summary>
-        /// <param name="settings">Settings to initialize the local audio track.</param>
-        /// <returns>Asynchronous task completed once the device is capturing and the track is created.</returns>
-        /// <remarks>
-        /// On UWP this requires the "microphone" capability.
-        /// See <see href="https://docs.microsoft.com/en-us/windows/uwp/packaging/app-capability-declarations"/>
-        /// for more details.
-        /// </remarks>
-        public static Task<LocalAudioTrack> CreateFromDeviceAsync(LocalAudioTrackSettings settings = null)
+        /// <param name="source">The track source which provides the raw audio frames to the newly created track.</param>
+        /// <param name="initConfig">Configuration to initialize the track being created.</param>
+        /// <returns>Asynchronous task completed once the track is created.</returns>
+        public static Task<LocalAudioTrack> CreateFromSourceAsync(AudioTrackSource source, LocalAudioTrackInitConfig initConfig)
```

Comment:
Does this still need to be async?

Reply:
No; that's one of the next reviews after this one :)